lib/src/parser.dart

```diff
@@ -66,6 +66,10 @@ class Parser {
       if (command != null) {
         validate(rest.isEmpty, 'Cannot specify arguments before a command.');
         var commandName = args.removeFirst();
+        if (command.isDeprecated) {
+          print('Deprecation Warning: ${command.deprecationMessage}');
```

Comment:
We should not `print` here, since we don't know that the normal output for `print` is what the author is using to communicate to the user.